lib/pure/math.nim

```diff
@@ -56,7 +56,7 @@ type
     fcNegInf     ## value is negative infinity
 
 proc classify*(x: float): FloatClass =
-  ## classifies a floating point value. Returns `x`'s class as specified by
+  ## Classifies a floating point value. Returns `x`'s class as specified by
```

Comment:
Dunno why we need to discuss it, you need to learn how to read. :P  "procedure classify of type xyz classifies a floating point value". See? Lowercase makes much more sense.